docs/docs/plugins.md

```diff
@@ -205,6 +205,18 @@ Here is as complete `manifest.json` example with all possible configurations:
       "permissions": ["moderator", "viewer"]
     }
   ],
+  "serverCommand": {
+    "chat": {
+      "sendPublicChatMessage": {
+        "allowedRoles": [
+          "all"
+        ]
+      },
+      "sendCustomPublicChatMessage": {
+        "allowedRoles": ["presenter", "moderator"] // "moderator","presenter", "all", "viewer"
+      }
+    }
+  }
```

Comment:
I don’t think `sendPublicChatMessage` requires permissions.
The goal is to restrict Custom Messages, those are the ones that could cause issues.
Public messages seem fine to send, since they can also be sent without relying on any plugin.